build/config/phone/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dom.presentation.discovery.enabled": true
```

Comment:
This setting is sync with gecko pref to turn on MDNS device discovery. Gaia can use settings to switch on/off this feature.